### PR TITLE
make the python module emit event data

### PR DESCRIPTION
**Pull request by fmoralesc**

### CODE REVIEW

File: src/python.rs [hunk 1]
```diff
@@ -7,12 +8,19 @@ fn to_html(_py: Python, buffer: String) -> PyResult<String> {
 }
 
 #[pyfunction]
-fn get_offsets(_py: Python, buffer: String) -> PyResult<()> {
+fn get_offsets(_py: Python, buffer: String) -> PyResult<&PyDict> {
```
Comment (alerque) [inline src/python.rs:11]: Why is this returning a reference `&PyDict` instead of just passing ownership of the dict (`PyDict`)?

**Review by alerque — CHANGES_REQUESTED** [review]
Don't mind the CI for now, it's not cooperating with the Rust build issues yet.

File: src/python.rs (hunk 1 above)
Comment (fmoralesc) [reply src/python.rs:11]: No good reason. This compiles and seems to work, and the alternative complains about the traits of the object.

Comment (fmoralesc) [reply src/python.rs:11]: What should be the approach to take here?

Comment (alerque) [reply src/python.rs:11]: Not your problem. This goes back to what I was complaining about when I couldn't combine the Lua & Python module generation without writing wrapper functions like this. It has to do with the pyo3 implementation.

**Review by alerque — APPROVED** [review]

File: src/python.rs (hunk 1 above)
Comment (fmoralesc) [reply src/python.rs:11]: Yes, this is the kind of issue I was complaining about too when I was forced to pass strings instead of structured data.

**Pull request merged**